client/components/webpack-build-monitor/index.jsx

```diff
@@ -5,7 +5,8 @@
  * External dependencies
  */
 import React from 'react';
-import { createStore } from 'redux';
+// eslint-disable-next-line wpcalypso/import-no-redux-combine-reducers
```

Comment:
is there a reason you disabled it instead of using the version of `combineReducers` in `state/utils`